test/agentchat/contrib/test_lmm.py

```diff
@@ -79,5 +79,53 @@ def test_print_received_message(self):
         self.agent._print_received_message.assert_called_with(message_str, sender)
 
 
+@pytest.mark.skipif(skip, reason="Dependency not installed")
```

Comment:
Should this test be added into the contrib openai test?

Reply:
It’s in contrib_lmm.yml . Should I combine with contribOpenAI?

Reply:
We separate tests that require OpenAI and that don't in different workflows.

Reply:
Here, I used "llm_config = False". So, I think it does not require actually calling OpenAI.

Reply:
@sonichi Should I resolve this issue here, and create another test file and PR to actually calling OpenAI?